Sources/TripKitUI/view model/TKUITripOverviewViewModel.swift

```diff
@@ -13,10 +13,11 @@ import RxCocoa
 
 import TripKit
 
-class TKUITripOverviewViewModel {
+class TKUITripOverviewViewModel: NSObject {
```

Comment:
Is this change still needed?

Reply:
Did something before, not needed anymore